data.json

```diff
@@ -0,0 +1,899 @@
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-LE-02.02",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-LE-03.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-LE-03.02",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-01.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-01.02",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-02.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "humanEvidence": {
+            "name": "manual-inspection",
+            "humanDescription": "Package manifest file (e.g., package.json) present in repo root."
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-04.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-05.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-05.02",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-VM-02.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "attestationRefEvidence": {
+            "name": "automated-scan",
+            "ref": {
```

Comment:
In the [Controls object in BaselinePredicate](https://docs.google.com/document/d/16zwe3eNwExvnaXLDUrkGNineZzB0r8arKE3G-cvNU0E/edit?tab=t.0), `ref` is intended purely for referring to another attestation (e.g. when merging attestations).  I think this shoud be an `ella`.

(Also, I'd skip the nesting of `attestationRefEvidence`)